docs/_writing_collections.rst

```diff
@@ -125,13 +125,22 @@ Processing Lists (Reductions)
 -------------------------------
 
 The ``data_collection`` parameter type can specify a ``collection_type`` or
-``list`` but whenever possible, it is actually better to not explicitly
+``list`` but whenever possible, it might actually be better to not explicitly
```

Comment:
This represented an IUC recommendation... I think I agree with the language changes here - but I was always more pro-collection inputs then everyone else. So I'm nervous about making this less of a recommendation.

Reply:
Agreed. I changed the wording to "it is recommended".